cardano-sl.cabal

```diff
@@ -12,17 +12,11 @@ build-type:          Simple
 extra-source-files:  README.md
 cabal-version:       >=1.10
 
-Flag with-web
-  default:     True
-  manual:      True
-
-  description: Build with web part.
-
 Flag with-wallet
```

Comment:
Why does this flag still exist? Is it used only to determine linker or for something else?